backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/housekeeping/HouseKeepingService.java

```diff
@@ -43,7 +43,7 @@ public class HouseKeepingService
 {
 	private static final Logger logger = LogManager.getLogger(HouseKeepingService.class);
 
-	public static final String SYSCONFIG_SKIP_HOUSE_KEEPING = "de.metas.housekeeping.SkipHouseKeeping";
+	public static final String SYSCONFIG_SKIP_HOUSE_KEEPING = "de.metas.housekeeping.SkipHouseKeeping.ApiAuditHouseKeepingTask";
```

Comment:
why changing the name of this "generic" sysconfig to something so specific?!

Reply:
fixed in https://github.com/metasfresh/metasfresh/pull/17557